src/components/db.ts

```diff
@@ -11,6 +11,10 @@ export const openDB = async (): Promise<Database> => {
       driver: sqlite3.Database
     });
     await db.run('CREATE TABLE IF NOT EXISTS saved_data (msg_id INTEGER PRIMARY KEY,data TEXT NOT NULL);');
+    await db.run(
+      'CREATE TABLE IF NOT EXISTS suggestions (suggestion_id INTEGER PRIMARY KEY NOT NULL, suggestion_author VARCHAR(255) NOT NULL,' +
+        'created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,suggestion VARCHAR(500) NOT NULL, suggestion_state VARCHAR(1) NOT NULL);'
```

Comment:
can we use full names for state instead of a single character?

Reply:
also I think we can just leave suggestion as `TEXT` instead of setting a limit on number of chars, unless you want to check the character limit on the application level as well